tox.ini

```diff
@@ -6,7 +6,7 @@ envlist =
        {py27,py32,py33,py34}-django{17,18,master}
 
 [testenv]
-commands = ./runtests.py --fast
+commands = ./runtests.py --fast {posargs}
```

Comment:
Sure i was just wondering how this related to the pull request 
Don't get me wrong i'm fine with it